graphene_django/converter.py

```diff
@@ -68,6 +69,29 @@ def description(self):
     return Enum(name, list(named_choices), type=EnumWithDescriptionsType)
 
 
+def generate_enum_name(django_model_meta, field):
+    if graphene_settings.DJANGO_CHOICE_FIELD_ENUM_CUSTOM_NAME and callable(
+        graphene_settings.DJANGO_CHOICE_FIELD_ENUM_CUSTOM_NAME
+    ):
+        name = graphene_settings.DJANGO_CHOICE_FIELD_ENUM_CUSTOM_NAME(field)
+    elif graphene_settings.DJANGO_CHOICE_FIELD_ENUM_V3_NAMING is True:
+        name = "DjangoModel{app_label}{object_name}{field_name}Choices".format(
```

Comment:
I don't think prepending `DjangoModel` to the enum name is good, it leaks the storage to the API.

I think it will be better just:

```suggestion
        name = "{app_label}{object_name}{field_name}Choices".format(
```

Reply:
Fair enough. Thanks @syrusakbary